### ISSUE TITLE
Fixed width format

### ISSUE THREAD

**Issue by gaborcsardi** [sindresorhus/pretty-bytes#58]
For progress bars, it would be nice to have an output format that is always the same width. E.g. a possible six character wide format:

```
            1 -> 0.0 kB
           10 -> 0.0 kB
          100 -> 0.1 kB
         1000 -> 1.0 kB
        10000 ->  10 kB
       100000 -> 100 kB
      1000000 -> 1.0 MB
     10000000 ->  10 MB
    100000000 -> 100 MB
   1000000000 -> 1.0 GB
  10000000000 ->  10 GB
 100000000000 -> 100 GB
1000000000000 -> 1.0 TB
```

**Comment by vladshcherbin** [comment]
@gaborcsardi maybe [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) can help you 🤔

**Comment by gaborcsardi** [comment]
Yeah, then I might just write another package, because it is not just a `padStart()` call, e.g. pretty-bytes returns `1 B` for 1 byte, and ideally I would line up both the units and the numbers. This is why I was suggesting to avoid `B` in the example above.

**Comment by vladshcherbin** [comment]
@gaborcsardi oh, I see, this wasn't obvious from the example above :)

**Comment by vladshcherbin** [comment]
@gaborcsardi for your case, another option is to use [filesize](https://www.npmjs.com/package/filesize) instead, you can set `symbols` there. A simple change of `B` to `B ` and `padStart` should make your example possible.

I switched to it because I need precision https://github.com/sindresorhus/pretty-bytes/issues/57 and it kinda works fine.

**Comment by Richienb** [comment]
@sindresorhus How do you want the fixed width to be achieved?

**Comment by sindresorhus** [comment]
Maybe instead we could just make add a `rightAlign` option.
